extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -0,0 +1,101 @@
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-user-notification"></a> `amp-user-notification`
+
+Displays a notification that allows the visitor of a site to give consent
+to be associated to a client identifier.
+
+#### Usage
+
+`amp-user-notification` requires 2 json endpoints which can be provided by
+the `data-show-if-href` and `data-dismiss-href` attributes. An `id` is required
+as this is transmitted to the endpoints as the json `elementId` field.
+
+To close `amp-user-notification` add a `on` attribute to a button with the
+following value scheme `on="event:id.dismiss"` (see example below).
+
+Example:
+
+```html
+<amp-user-notification
+    layout=container
+    id="amp-user-notification1"
+    data-show-if-href="http://localhost:8000/test/manual/show-dialog.json"
+    data-dismiss-href="http://localhost:8000/api/echo/post">
+    This site uses cookies to personalize content.
+    <a href="">Learn more.</a>
+   <button on="tap:amp-user-notification1.dismiss">I accept</button>
+</amp-user-notification>
+```
+
+#### Attributes
+
+**data-show-if-href** (Required)
+
+endpoint where `amp-user-notification` will execute a
+`GET` request to (once the document is active).
+
+ - Request data on `GET`
+    ```json
+    { "elementId": "id-of-amp-user-notification", "externalCID": "cidstring" }
+    ```
+
+ - Response data should be in json with a `show_notification` attribute
+   that is of `boolean` type.
+
+    ```json
+    { "show_notification": true }
+    ```
+
+**data-dismiss-href** (Required)
+
+endpoint where `amp-usernotification` will execute a
+`POST` request to once the visitor of the website has explicitly agreed.
+
+  - `POST` data
+
+    ```json
+    { "elementId": "id-of-amp-user-notification" }
+    ```
+
+  - Response should be a 200 HTTP code and no data is expected back.
+
+**credentials**
+
+defines a `credentials` option as specified by the
+[Fetch API](https://fetch.spec.whatwg.org/). To send credentials, pass the
+value of "include".
+
+#### Behavior
+
+`amp-user-notification` initiates the `GET` request to `data-show-if-href`
+only if the current document is active. The component will only be visible
+if `data-show-if-href` endpoint returns with `show_notification: true`
+else with `show_notification: false` it is assumed that consent is given and
+variable substition will occur on `CLIENT_ID`.
+
+If `amp-user-notification` is displayed variable substitution on `CLIENT_ID`
+will never occur unless the visitor of the website agrees to do so.
+
+Once the visitor dismisses the notification the `POST` will occur on the
```

Comment:
I think you can basically kill this entire section if you are more elaborate above.